posthog/test/base.py

```diff
@@ -694,22 +708,25 @@ def wrapped(self, *args, **kwargs):
     return wrapped
 
 
-def also_test_with_different_timezone(fn):
+def also_test_with_different_timezones(fn):
     """
-    Runs the test twice, including in a timezone other than UTC to catch timezone handling bugs.
+    Runs the test twice: once with the project in the default UTC timezone, then again with a negative UTC timezone.
+    This is intended for catching bugs around timezone-awareness.
     """
```

Comment:
Made this decorator more robust - now we're stretching timezone handling in all directions